test/e2e/helpers/wallet/wallet-api.ts

```diff
@@ -106,12 +106,16 @@ export class APIHelper {
     passphrase = 'password1',
     walletName = 'Wallet 1',
     keyName = 'Key 1',
-    telemetryOptIn = false
+    telemetryOptIn = false,
+    recoveryPhrase = ''
```

Comment:
nitpick

```suggestion
    recoveryPhrase = undefined
```

Reply:
this broke the code on its own so made it string | undefined